depscan/cli.py

```diff
@@ -549,10 +549,12 @@ async def run_scan():
     """
     q = request.args
     params = await request.get_json()
+    uploaded_bom_file = await request.files
+
     url = None
     path = None
-    multi_project = None
-    project_type = None
+    multi_project = "false"
+    project_type = "universal"
```

Comment:
universal type would take a long time. So best to mandate a type in server mode

Reply:
As a cli, universal is fine. But as a server, we can run it for a specific type.